Resources_details.md

```diff
@@ -0,0 +1,51 @@
+### Setup
+
+ * List of projects per Research Program
+ * Use CMS wiki or my.nci.org.au to find name of project coordinator
+ * Information to give to the project coordinator before joining (CI or delegate)
+ * Request to join the project through my.nci.org.au
+ * Determining resources required, e.g. running model, analysing data
+    - Compute resources (KSU)
+    - Storage resources (GB): short term (`/short`), medium term (`/gdata`) and archive (`mdss`)
+ * Use of `nci_account`
+
+[Link to training](???)
+
+### Monitor resource use
+
+ * Review of the tools available to you
+    * nci_account 
+    * <fsystem>_files_report
+    * lquota
+    * nci_monitor
+    * du
+    * ncdu
+    * find
+
+[Link to training](???)
+
+### Managing storage
+ * Managing file ownership and permissions:
+    * chmod
+    * chgrp
+    * ACLs
+ * File compression:
+    * nccompress
+    * cdo
+    * nco
+    * gzip, zip, bzip
+ * Prepare data for safe keeping over medium term or longer:
+   * what can be deleted
+        * Use of `find` to automate the process
+   * documentation
+   * tarring
+        * tar
+        * `find` to automate the process
+ *  transfer:
+   * difference between cp and mv
+   * rsync
+   * scp
+   * sftp
+   * mdssdiff
```

Comment:
Do we forget `netmv` and the other one (forgot the name) ? Find by me. Just noting they aren't there.